app/controllers/templates_controller.rb

```diff
@@ -99,6 +99,10 @@ def export
     send_data @template.to_erb, :type => 'text/plain', :disposition => 'attachment', :filename => @template.filename
   end
 
+  def resource_class
```

Comment:
actually this is no longer necessary, but i think it makes sense to leave it here anyways for consistency with the default.